tests/test_chatbot.py

```diff
@@ -0,0 +1,33 @@
+import os
+from pathlib import Path
+
+from buster.chatbot import Chatbot, ChatbotConfig
+
+TEST_DATA_DIR = Path(__file__).resolve().parent / "data"
+DOCUMENTS_FILE = os.path.join(str(TEST_DATA_DIR), "document_embeddings_huggingface_subset.tar.gz")
```

Comment:
This will break every time we change the way we store data, which is often since we're in the middle of a refactor. The proper way would be to create a mock file in a fixture.

Reply:
ok, good idea, i can create a mock file but it will have to create the embeddings every time, not a big deal but i'll need a quick script from you

Reply:
Do we need real embeddings though? We can (should?) monkeypatch `get_embedding` and `self.documents.retrieve`, since we're testing the chatbot.py code, not the rest.

Reply:
Do we need real embeddings though? We can (should?) monkeypatch `get_embedding` and `self.documents.retrieve`, since we're testing the chatbot.py code, not the rest.

Reply:
the original intent of this one specific test in this PR was to have it be as close as possible to the actual chatbots such taht when we propose new changes, we are fairly confident things didn't break in our deployed apps. I do agree that we need a whole suite of tests, but this one is just the "bare minimum" to ensure things work

Reply:
then let's just merge it and fix later